lib/PuppeteerSharp/Frame.cs

```diff
@@ -92,6 +98,12 @@ internal Task<ElementHandle> AddScriptTag(dynamic options)
             throw new NotImplementedException();
         }
 
+        internal async Task<string> GetTitleAsync()
+        {
+            var result = await EvaluateAsync("document.title");
```

Comment:
using `() => document.title` didn't work since we are missing the logic - https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/lib/ExecutionContext.js#L53

Reply:
Evaluate is working? 😱

Reply:
just for expressions for now, for functions we need a bit more work